src/core/constraint.jl

```diff
@@ -189,3 +202,14 @@ function constraint_storage_loss(pm::GenericPowerModel, n::Int, i, bus, r, x, st
     sd = var(pm, n, :sd, i)
     JuMP.@NLconstraint(pm.model, ps + (sd - sc) == standby_loss + r*(ps^2 + qs^2)/vm^2)
 end
+
+function constraint_storage_on_off(pm::GenericPowerModel, n::Int, i, charge_lb, charge_ub, discharge_lb, discharge_ub)

```

Comment:
This constraint should also force `ps` and `qs` to zero as well.  There will need to be a specialized version for the  version for the `AbstractActivePowerFormulation` type as `qs` is not defined in those models.

Reply:
If ps=0, sd=0, sc=0, and qs=0 then standby_loss also must equal 0. 

Looking at the storage energy constraint 
https://github.com/lanl-ansi/PowerModels.jl/blob/master/src/core/constraint.jl#L173

```julia
JuMP.@constraint(pm.model, se_2 - se_1 == time_elapsed*(charge_eff*sc_2 - sd_2/discharge_eff))
```

I think it makes sense to set ps and qs to zero, but leave sc and sd active to represent the storage standby loss.

Reply:
Good point.  I think that's the correct semantics in the a non-linear model.  In the special case of the a lossless model we can also put `sc` and `sd` to zero, which will strengthen the formulation.